README.md

```diff
@@ -77,6 +77,11 @@ We strongly recommend following this structure and making your extensions so tha
 dependent on anything other than `solidus_core`, only augmenting the functionality of the other
 engines when they are available.
 
+Any decorators placed in `app/decorators` will also be loaded in, and should contain the type
+of object being decorated in the file structure. For instance, if you're decorating the variant
+model, you should structure the file like this:
+`app/decorators/models/#{extension_name}/spree/variant_decorator.rb`
```

Comment:
Is there a benefit to having the extension name in there? It feels like it makes the path longer and it's already inside the extension.

Reply:
@jarednorman the reason for doing that is to namespace each extension's decorators. Otherwise, if two extensions decorate the same class, the decorator modules get mixed up.

Reply:
Also, if you analyze the ancestor of a class, it's useful to see where each module are included from.

Reply:
Right right, makes sense.